cli/src/pcluster/validators/iam_validators.py

```diff
@@ -57,6 +59,28 @@ def _validate(self, policy: str):
             self._add_failure(str(e), FailureLevel.ERROR)
 
 
+class IamResourcePrefixValidator(Validator):
+    """
+    Iam Resource Prefix Validator.
+
+    Verify if the Resource Prefix is compliant with IAM naming conventions
+    """
+
+    IAM_PATH_NAME_REGEX = r"^((/[a-zA-Z0-9_.,+@=-]+)+)/"
+    IAM_ROLE_NAME_PREFIX = r"^[a-zA-Z0-9_.,+@=-]+$"
```

Comment:
It's not just a role name prefix, but also policy name prefix, right?

Reply:
Will rename this to IAM_NAME_PREFIX_REGEX